src/calculator.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import CalculatorTitle from './calculatorTitle';
+import OutputScreen from './outputScreen';
+import Button from './button';
+
+const Calculator = () => (
+  <div className="frame">
+    <CalculatorTitle value="Calculator" />
+    <div className="mainCalc">
+      <OutputScreen />
+      <div className="button-row">
+        <Button label="Clear" />
+        <Button label="Delete" />
```

Comment:
Kindly create calculator panel with all the buttons in the following groups: 👇 

![image](https://user-images.githubusercontent.com/31897434/130065243-af3f9b47-8f7f-4e63-85b6-6dc22b095e37.png)

🤔 `Clear` and `delete` should not be inside the group.